Entity Explorer - Account.ipynb

```diff
@@ -6,20 +6,23 @@
    "source": [
```

Comment:
<p>I find this a bit confusing as its not clear immediately what the difference between "AzureActiveDirectory" and "OfficeActivty" is here - Account types doesn't adequately describe what this is. I would consider adding some more markdown that explains this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>